media/client/main/source/WebAudioPlayer.cpp

```diff
@@ -184,7 +166,7 @@ bool WebAudioPlayer::writeBuffer(const uint32_t numberOfFrames, void *data)
         return false;
     }
 
-    uint8_t *shmBuffer = m_sharedMemoryManager->getSharedMemoryBuffer();
+    uint8_t *shmBuffer = m_clientController.getSharedMemoryBuffer();
```

Comment:
Should we listen for state change to protect this write like do in MediaPipeline here? There's nothing stopping the shm been terminated while this write is ongoing. 

Wasn't done previously, doesn't have to be done this this delivery.

Reply:
Agree, I will do it in next commit, as it has to be tested well.